Makefile

```diff
@@ -172,6 +172,15 @@ dm_integration_test_build: tools_setup
 	$(FAILPOINT_DISABLE)
 	tests/prepare_tools.sh
 
+dm_integration_test_build_worker: tools_setup
```

Comment:
It is a bit strange if there is only `dm_integration_test_build_worker` but no `dm_integration_test_build_master` 😂

Reply:
it's convenient for local test, if someone often needs to change/build dm-master , he can add `dm_integration_test_build_master` 😂